components/chat/ChatArea/index.tsx

```
@@ -0,0 +1,171 @@
+      content: "",
+      role: "assistant",
+    });
+
+    requestAnimationFrame(() => {
+      scrollContainerRef.current?.scrollTo({
+        behavior: "smooth",
+        top: scrollContainerRef.current?.scrollHeight,
+      });
+    });
+
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)